packages/components/src/index.ts

```diff
@@ -12,7 +12,11 @@ export {
 } from '@wordpress/primitives';
 
 // Components.
-export { default as __experimentalAlignmentMatrixControl } from './alignment-matrix-control';
+export {
+	/** @deprecated Import `AlignmentMatrixControl` instead. */
```

Comment:
Do we plan on adding a deprecation warning in a follow-up once all instances are migrated, or are we keeping it "soft deprecated" for now?

Reply:
I'm thinking we can decide on a per-component basis, using reasoning from #61099. Maybe we'll want to do a more aggressive cleanup in the future, but I don't think it's a pressing need for the moment. I'm anticipating most of these to be passive deprecations.